hack/test/deploy/scenarios/wavefront-advanced.yaml

```diff
@@ -0,0 +1,203 @@
+# Need to change YOUR_CLUSTER_NAME and YOUR_WAVEFRONT_URL accordingly
+apiVersion: wavefront.com/v1alpha1
+kind: Wavefront
+metadata:
+  name: wavefront
+  namespace: wavefront
+spec:
+  clusterName: YOUR_CLUSTER_NAME
+  wavefrontUrl: YOUR_WAVEFRONT_URL
+  wavefrontTokenSecret: wavefront-secret
+  dataCollection:
+    metrics:
+      enable: true
+      customConfig: wavefront-collector-config
+      clusterCollector:
+        resources:
+          requests:
+            cpu: 200m
+            memory: 10Mi
+          limits:
+            cpu: 400m
+            memory: 512Mi
+      nodeCollector:
+        resources:
+          requests:
+            cpu: 200m
+            memory: 10Mi
+          limits:
+            cpu: 200m
+            memory: 256Mi
+  dataExport:
+    wavefrontProxy:
+      enable: true
+      metricPort: 2878
+      preprocessor: advanced-proxy-preprocessor-rules
+      resources:
+        requests:
+          cpu: 200m
+          memory: 1Gi
+        limits:
+          cpu: 500m
+          memory: 3Gi
+      tracing:
+        wavefront:
+          port: 30000
+          samplingRate: "0.1"
+          samplingDuration: 100
+        zipkin:
+          port: 9411
+          applicationName: "zipkin"
+        jaeger:
+          port: 30001
+          httpPort: 30080
+          grpcPort: 14250
+          applicationName: "jaeger"
+      histogram:
+        port: 40000
+        minutePort: 40001
+        hourPort: 40002
+        dayPort: 40003
+#      httpProxy:
```

Comment:
Should we remove the commented httpProxy sections?

Reply:
yes, thank you.  I will remove it